tests/e2e/ans/publishANSContracts.ts

```diff
@@ -0,0 +1,106 @@
+import { execSync } from "child_process";
```

Comment:
lets put this file under `~/scripts/` , we would need to run it before all tests

Reply:
@gregnazario improved the flow here! [He moved the invocation of the script to the e2e test file itself](https://github.com/aptos-labs/aptos-ts-sdk/blob/02173dbf29a168564b8ab587aa3a3fc1675703d9/tests/e2e/ans/ans.test.ts#L24-L25). Now, whenever this test runs it will ensure it has a published contract, regardless of the command used to make it run. Keeping it colocated to the test makes sense with this change!

Reply:
how then you promise that this file runs before the actual `ans.test.ts` file?

Reply:
ohh I see you run it in beforeAll in the other test, got it.

Reply:
Tried to put it under scripts... it was failing with some really messy esm vs cjs problems, so this keeps it compatible with both, but it just does it before the test (and only once)